packages/react/src/useNhostBackendUrl.ts

```diff
@@ -3,6 +3,7 @@ import { useContext } from 'react'
 import { NhostReactContext } from './provider'
 
 /**
+ * @internal @deprecated Nhost services can now be split to subdomains to improve performance with CDN. Use `nhost.auth.url` or `nhost.storage.url` instead.
```

Comment:
Is it necessary to exclude this deprecation note from the auto-generated docs?

Reply:
Not really. I'm not just fond of having too much deprecated contents in our documentation - users already have inline documentation within their IDE

Reply:
Yep, that's true, but we should be consistent with this IMO. We should get rid of deprecated functionality in a later release anyways.